engineyard/ey-cookbooks-stable-v5
# CC 1057 EFS

Description of your patch
--------------

Adds EFS Support for Stable V5


Recommended Release Notes
--------------
This update allows EFS to be mounted automatically by the cookbooks

Estimated risk
--------------

Low


Components involved
--------------

EFS

Description of testing done
--------------

1. Boot stack under V5 with feature enabled in a vpc in US West 2, US East 1, EU West 1
1. Verify chef runs correctly
1. Provision EFS Cluster and wait for it to provision
1. Run an `Apply` and Verify chef runs correctly
1. Verify Chef runs correctly
1. Verify /efs exists with output of `df -h` and /efs exists in both places
1. Remove efs cluster and run chef and verify runs correctly

QA Instructions
--------------

1. Boot stack under V5 with feature enabled in a vpc in US West 2, US East 1, EU West 1
1. Verify chef runs correctly
1. Provision EFS Cluster and wait for it to provision
1. Run an `Apply` and Verify chef runs correctly
1. Verify Chef runs correctly
1. Verify /efs exists with output of `df -h` and /efs exists in both places
1. Remove efs cluster and run chef and verify runs correctly
